pkg/plugin/extension.go

```diff
@@ -72,6 +74,11 @@ func (w *Extension) Visualizer() visualizer.Visualizer {
 	return w.visualizer
 }
 
+// WidgetMetaData _
+func (w *Extension) WidgetLayout() *scene.WidgetLayout {
```

Comment:
```suggestion
func (w *Extension) Layout() *scene.WidgetLayout {
```

`widget.WidgetLayout()`, the widget is repeated twice. `widget.Layout()` conveys the meaning well enough.